QualityPolicy.md

```diff
@@ -27,8 +26,7 @@
 - [ ] US acceptance criteria are met
 - [ ] Static analysis checks pass
 ```
-* Period branches being merged into the Main branch shall be reviewed by 3 group members.
 
 ## Releases
-* A Release Tag shall be created for the state of the Main branch at the end of each Period
-* The Release Tag shall follow the naming scheme `Period-X-Release`
\ No newline at end of file
+* A Release Tag shall be created for the state of the Main branch after each merged pull request
+* The Release Tag shall follow the naming scheme `#.#.#`
```

Comment:
Should we include rules for naming the release tag? IIRC, it was last number for user stories, second number for period releases?

Reply:
The naming rules you mentioned are now added.